.travis/build_package.sh

```diff
@@ -8,10 +9,10 @@ function build_examples {
   mkdir -p build-travis
   cd build-travis
 if [ $IS_WINDOWS ]; then
-  cmake -DCGAL_DIR="/usr/local/lib/cmake/CGAL" -DCMAKE_CXX_FLAGS_RELEASE="${CXX_FLAGS}" -G "Visual Studio 12 2013 Win64" ..
+  cmake -DCGAL_DIR="C:\projects\cgal\build" -DCMAKE_CXX_FLAGS_RELEASE="${CXX_FLAGS}" -G "Visual Studio 12 2013 Win64" ..
   cmake --build .
 else
-  cmake -DCGAL_DIR="/usr/local/lib/cmake/CGAL" -DCMAKE_CXX_FLAGS_RELEASE="${CXX_FLAGS}" ..
+  cmake -DCGAL_DIR="C:\projects\cgal\build" -DCMAKE_CXX_FLAGS_RELEASE="${CXX_FLAGS}" ..
```

Comment:
In the `else`, `[ $IS_WINDOWS]` returns false, so you should assume the platform is Linux (Travis), and the `CGAL_DIR` should be the path for CGAL on Travis.